frontend/.nvmrc

```diff
@@ -1 +1 @@
-v15
+v16.14.2
```

Comment:
Pinning to  avoid GH Action lint error exit code 243
See: https://stackoverflow.com/a/71892226/3120863